_kbarticles/2022-12-08-common-kibana-searches-on-clouddotgov.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title: "Common Kibana Searches on cloud.gov"
+date: December 8, 2022
+excerpt: This article documents a brief explaination of what Kibana does and common Kibana search queries on the cloud.gov platform.
+
+---
+
+## What does Kibana do?
+
+Application logs are visualized via Kibana, which has a [user guide](https://www.elastic.co/guide/en/kibana/current/index.html) that explains more about how to use it and customize your views.
+
+## What are some types of logs?
+
+Cloud Foundry assigns a type to each log message depending on its origin. Application logs are assigned the APP log type.  HTTP requests being routed to an app will produce the RTR log type. The various types of logs are listed in the documentation [here](https://docs.cloudfoundry.org/devguide/deploy-apps/streaming-logs.html#format).
+
+## How to chart the information:
+
+Spikes in traffic can be visualized by using the chart provided in Kibana. To find the specific information you are requesting you need to use the Discover tab to search RTR logs, and then create the chart based on the saved search. Use a count aggregation as the Y-Axis and a date histogram aggregation for the X-Axis.
```

Comment:
I think it would be pretty helpful to create a screenshot showing how to configure this type of visualization. And then a screenshot of the end result. 

Use a test app to create the visualization and screenshot, not any customer app, so that we don't expose customer information

I can pair on this with you if you need help

Reply:
Mark, Thank you for your help with this! Added the images.